tests/unit/syncModel/workflowTransformer.test.ts

```diff
@@ -0,0 +1,179 @@
+import { CollectionContracts, ContentTypeContracts, WorkflowContracts } from "@kontent-ai/management-sdk";
+import { describe, expect, it } from "vitest";
+
+import { EnvironmentModel } from "../../../src/modules/sync/generateSyncModel.ts";
+import { transformWorkflowModel } from "../../../src/modules/sync/modelTransfomers/workflow.ts";
+
+describe("transformWorkflowModel", () => {
+  it("transforms workflows correctly", () => {
+    const contentTypes = [
+      {
+        id: "1cdb6e21-3330-4f0f-88cd-171098950e4f",
+        name: "type 1",
+        codename: "type_codename_1",
+        elements: [
+          {
```

Comment:
you probably don;t need those elements here :D

Reply:
copypasted from another test and not adjusted accordingly :)